Keep Mycroft logs from VK test runs

krisgesling opened the pull request:
This extracts the standard files from /var/log/mycroft from the Docker container after the Voight Kampff test runs to provide extra debugging information. Particularly useful if the issue being reported cannot be replicated locally as we've seen with the stop method on the News and Singing Skills.

Currently the links for the log files are provided only in the failure email. Presumably there's no need to see the logs if everything passed.

### To test
See the [Jenkins console for PR-1360](https://hudson.mycroft.team/job/mycroft-skills/view/change-requests/job/PR-1360/22/console) that got replayed with this configuration, and check the related email from that job.

forslund commented on Jenkinsfile:
```diff
@@ -1,189 +1,213 @@
-            when {
-                anyOf {
-                    changeRequest()
+    pipeline {
```
Is the indentation change intended?

krisgesling replied:
Nope, fixed now.

forslund reviewed (commented):
Looks good, the complete indentation change makes it hard to review since all lines are reported as changed.

But the log volume looks Docker arguments correct and the logfiles are removed after each job...
Perhaps add the link to the logs to the VK test PR comment?

forslund commented:
Also seems like an old commit tagged along on this PR...

krisgesling commented:
All fixed now. 
- rebased off 20.08 branch
- fixed the indentation
- added log links to PR comment

I've also triggered one more replay of PR-1360 using this Jenkinsfile as a final test:
https://hudson.mycroft.team/job/mycroft-skills/view/change-requests/job/PR-1360/27/

devops-mycroft commented:
Voight Kampff Integration Test Succeeded  ([Results](https://reports.mycroft.ai/skills/PR-1391))

forslund commented:
Looks good! Did you find out why Jenkins reported unstable?

krisgesling commented:
Somehow it had a single test from the Web-Launcher Skill that failed. Not sure why:
1. the test was included at all
2. it reported unstable instead of fail

That Skill isn't mentioned at all in the [console output](https://hudson.mycroft.team/job/mycroft-skills/job/PR-1391/1/console), so I'm wondering if the docker image didn't get fully cleaned from the last run or something.
